querybean-generator/src/main/java/io/ebean/querybean/generator/ModuleMeta.java

```diff
@@ -1,21 +0,0 @@
-package io.ebean.querybean.generator;
-
-import java.util.List;
-
-class ModuleMeta {
```

Comment:
How did this get removed?

Reply:
because we can extract the data directly from the annotation instead of putting it in a proxy class